themes/mms-onprem/page.html

```diff
@@ -96,7 +96,7 @@ <h4 class="alert-heading" style="margin-bottom: 10px;">The MongoDB Agent has bee
         {% for v in theme_version_selector %}
           {% if not v.current %}
           <li>
-            <a href="#" data-path="{{ v.path }}" class="is-on-prem mms-version-selector">
+            <a href="#" data-path="{{ 'docs/' + v.path }}" class="is-on-prem mms-version-selector">
```

Comment:
No idea if this codepath is still in use, tried to find where it existed but couldn't 🤷‍♂️